Apps/AdminWebClient/src/Server/Models/AdminConfiguration.cs

```diff
@@ -47,5 +47,10 @@ public class AdminConfiguration
         /// Gets or sets the windows timezone id.
         /// </summary>
         public string WindowsTimeZoneId { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets the minimum number of notes to be counted.
+        /// </summary>
+        public int MinimumNotesCount { get; set; } = 2;
```

Comment:
q: should this be in the appsettings.json file?

Reply:
it loads from there, this is just a default value.